modules/configuring-firewall.adoc

```diff
@@ -40,7 +40,11 @@ If your environment has a dedicated load balancer in front of your {product-titl
 
 |`access.redhat.com` ^[1]^
 |443
-|Hosts all the container images that are stored on the Red Hat Ecosytem Catalog, including core container images.
+|Hosts all the container images that are stored on the Red Hat Ecosystem Catalog, including core container images.
```

Comment:
We should change this description for `access.redhat.com` to distinguish it from `registry.access.redhat.com`. 
It seems `access.redhat.com` holds the signature store while `registry.access.redhat.com` holds the registry. 
So I would change this description to say:

```suggestion
|Hosts a signature store that a container client requires for verifying images pulled from `registry.access.redhat.com`.
```

Reply:
Fixed. Thanks.